handlers/discount-api/test/applyDiscountIntegration.test.ts

```diff
@@ -84,12 +84,29 @@ test('digi subs can have a discount but dont get an email', async () => {
 		today,
 	);
 
+	const paymentDate = today.add(16, 'day');
 	const expected: ApplyDiscountResponseBody = {
-		nextNonDiscountedPaymentDate: zuoraDateFormat(today.add(16, 'day')),
+		nextNonDiscountedPaymentDate: zuoraDateFormat(paymentDate),
+	};
+
+	const expectedEmail: EmailMessageWithUserId = {
+		To: {
+			Address: 'test.user@thegulocal.com',
+			ContactAttributes: {
+				SubscriberAttributes: {
+					first_name: 'Test',
+					last_name: 'User',
+					first_discounted_payment_date: paymentDate.format('DD MMMM YYYY'),
+					next_non_discounted_payment_date: paymentDate.format('DD MMMM YYYY'),
```

Comment:
FIXME once the code is fixed the test needs changing accordingly
```suggestion
					next_non_discounted_payment_date: paymentDate.add(3, 'month').format('DD MMMM YYYY'),
```